tests/core/pyspec/eth2spec/test/helpers/constants.py

```diff
@@ -14,25 +14,25 @@
 SHARDING = SpecForkName('sharding')
 CUSTODY_GAME = SpecForkName('custody_game')
 DAS = SpecForkName('das')
-EIP4844 = SpecForkName('eip4844')
+DENEB = SpecForkName('deneb')
```

Comment:
This is line that is giving us problems in the CI. Seems like a caching problem because this is correct

cc: @hwwhww

Reply:
summon @parithosh  (not in a hurry!)